src/main/java/io/rcktapp/api/handler/sql/SqlDb.java

```diff
@@ -283,11 +289,10 @@ public void reflectDb() throws Exception
          return;
       }
 
+      Executor dbMetadataExecutorPool = Executors.newFixedThreadPool(50);
```

Comment:
This thread pool is used to parallelize the database reflection metadata retrieves that were serialized in this code.